core/src/main/java/org/apache/carbondata/core/datastore/chunk/impl/DimensionRawColumnChunk.java

```diff
@@ -92,8 +93,10 @@ public DimensionColumnPage decodeColumnPage(int pageNumber) {
    * @param index
    * @return
    */
-  public DimensionColumnPage convertToDimColDataChunkWithOutCache(int index) {
+  public DimensionColumnPage convertToDimColDataChunkWithOutCache(int index,
+      boolean isLongStringColumn) {
```

Comment:
why this is required???...as mentioned in earlier comment...reader does not know which type of data it is reading

Reply:
What does `reader` mean?